### DISCUSSION TITLE
Customising plugin icons

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [anyrun-org/anyrun#80] [Q&A] [answered]
Is it possible to customise the icons displayed for the plugin? 

![image](https://github.com/Kirottu/anyrun/assets/20235646/254ba714-e4e0-46d1-a9cd-ad19f981db13)

Talking about the icons that appear to the left of the plugin title.

**Comment by Kirottu** [comment, marked as answer]
The plugin icons are set by the plugin itself, and right now at least for the plugins in the Anyrun repo are not configurable.
